funnel/models/project.py

```diff
@@ -549,16 +557,14 @@ def roles_for(self, actor=None, anchors=()):
         Project, lazy='dynamic',
         primaryjoin=db.and_(
             Profile.id == Project.profile_id, Project.parent_id == None,
-            Project.state.PUBLISHED),
-        order_by=Project.date.desc())  # NOQA
+            Project.state.PUBLISHED))  # NOQA
 
 
 Profile.draft_projects = db.relationship(
         Project, lazy='dynamic',
         primaryjoin=db.and_(
             Profile.id == Project.profile_id, Project.parent_id == None,
-            db.or_(Project.state.DRAFT, Project.cfp_state.DRAFT)),
-        order_by=Project.date.desc())  # NOQA
+            db.or_(Project.state.DRAFT, Project.cfp_state.DRAFT)))  # NOQA
```

Comment:
We should start dropping `# NOQA` in query statements by switching to `column.is_(None)` and `column.isnot(None)`